CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ### Features
 
+* Add [vLLM](https://github.com/vllm-project/vllm) backend to serve, chat and generate commands.
+* Add `--backend` flag to `ilab model serve` command to allow for specifying the backend to use
+   when serving a model. This is useful when you have multiple backends installed and want to
+   specify which one to use. Currently, the only supported backend is `llama-cpp`.
```

Comment:
@leseb This should include `vllm` as a supported backend, right?

Reply:
updating